test/utils/configuration/configuration.go

```diff
@@ -56,8 +60,26 @@ func init() {
 
 	Platform, err = detect.Detect()
 	if err != nil {
-		panic(fmt.Errorf("failed to detect a platform: %w", err))
+		Platform = getPlatformFromEnv(envVarPlatform)
+	}
+	if Platform == platform.Unknown {
```

Comment:
why is it important to check the platform?

Reply:
it's because the install/uninstall suites. the `test/utils/configuration` package is used in these but also in the `serial` suite, so we need this fix. Without, we get
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift-kni_cnf-features-deploy/1010/pull-ci-openshift-kni-cnf-features-deploy-master-ci/1502239721626013696
```
running bin/cnftests -junit _cache/junit -ginkgo.dryRun -ginkgo.v
I0311 11:14:02.441403   22177 request.go:665] Waited for 1.046519169s due to client-side throttling, not priority and fairness, request: GET:[https://172.30.0.1:443/apis/console.openshift.io/v1?timeout=32s](https://172.30.0.1/apis/console.openshift.io/v1?timeout=32s)
panic: failed to detect a platform: clusterversions.config.openshift.io is forbidden: User "system:serviceaccount:ci-op-x0q88vkn:default" cannot list resource "clusterversions" in API group "config.openshift.io" at the cluster scope
goroutine 1 [running]:
github.com/openshift-kni/numaresources-operator/test/utils/configuration.init.0()
	/go/src/github.com/openshift-kni/cnf-features-deploy/vendor/github.com/openshift-kni/numaresources-operator/test/utils/configuration/configuration.go:59 +0x105
/go/src/github.com/openshift-kni/cnf-features-deploy
```

Reply:
ok thanks for clarifying that.